Python/getopt.c

```diff
@@ -62,7 +62,7 @@ int _PyOS_GetOpt(Py_ssize_t argc, wchar_t * const *argv, int *longindex)
     wchar_t *ptr;
     wchar_t option;
 
-    if (*opt_ptr == '\0') {
```

Comment:
Why this change?

Reply:
This is unrelated to the issue at hand, so please revert.

Reply:
I always want to make some almost useless modification to the source code when I  study  the CPython internals. If I submit a PR for every modification, it will waste a lot of time. So I will close this PR.  I hope I will make some substantial contribution to CPython in the near future. Thank you for your time !